### PR TITLE
Add user attribute to feedback (when using) - tiny PR

### PR DESCRIPTION
**Pull request by karenav**

### CODE REVIEW

**Review by tbarnahor — APPROVED** [review]

File: project/src/main/java/com/google/sps/data/DataAccessor.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public DataAccessor() {
   *     previously added to datastore.
   */
   public boolean isRegistered(String userId) {
-    checkArgument(!Strings.isNullOrEmpty(userId), "User ID may not be null or empty");
+    checkArgument(!isNullOrEmpty(userId), "User ID may not be null or empty");
```
Comment (szwbar) [inline project/src/main/java/com/google/sps/data/DataAccessor.java:57]: Is this change intended?

**Review by szwbar — APPROVED** [review]
Note that your PR description is incorrect or at least lacking, since you added this only in tests.

File: project/src/main/java/com/google/sps/data/DataAccessor.java (hunk 1 above)
Comment (karenav) [reply project/src/main/java/com/google/sps/data/DataAccessor.java:57]: Yes
